src/main/java/carRacing/model/RacingCar.java

```diff
@@ -3,14 +3,18 @@
 public class RacingCar {
 
     private int position;
+    private String name;
 
     public RacingCar() {
+        this.position = 0;
+    }
 
+    public RacingCar(String name){
         this.position = 0;
+        this.name = name;
     }
 
     public int positionOfRacingCar() {
```

Comment:
```suggestion
    public int position() {
```
어차피 RacingCar에게 보내는 메시지인데 RacingCar를 또 언급할 필요는 없어보여요